pkg/controller/apply.go

```diff
@@ -47,8 +45,7 @@ func (c *Controller) Apply(ctx context.Context, res *Resource) error {
 
 	l.Info("apply changing", "key", key, "kind", resource.GetObjectKind().GroupVersionKind())
 
-	if err := c.Client.Patch(ctx, resource, client.Apply, &client.PatchOptions{
-		Force:        &force,
+	if err := c.Client.Update(ctx, resource, &client.UpdateOptions{
```

Comment:
So switch back to update and then append a new mutation to fix the problems caused by using update? @cndoit18

Reply:
> 

right